source/instructors/admin/organization/sso.rst

```diff
@@ -0,0 +1,108 @@
+.. meta::
+   :description: You can enable SSO integration to allow you to configure SSO service provider for your users to login through your SSO provider
+   
+.. _sso-integration:
+
+Enable SSO Integration
+======================
+
+You can enable sso integration to integrate with your own SSO service provider from the **Organization > Integrations** page in Codio.
+
+    .. image:: /img/sso-integration.png
+       :alt: SSO Integration
+       
+.. Important:: Your SSO Custom User fields need to be set up with **FirstName**, **LastName**, **Email Address** and **Role** (Teacher/Mentor or Student).
+
+Enable SSO
+----------
+
+1: Enable SSO and select the SSO type. Currently suported **SAML 2.0**
+
+2: Log in to your SSO service provider
+
+3: Find and add the SAML application 
+
+4: Go to **Configuration** and copy in from Codio
+
+    - the **Login URL**
+    - the **ACS (CONSUMER) URL**
+    - the **ACS (CONSUMER) URL VALIDATOR**
+    
+5: Save the configuration
+
+6: Go to **Parameters** and set up the Custom User fields, checking **Include in SAML assertion** mapping the value field to the relevant fields from your SSO
+
+    - for **Role**, set the value to **Role(Custom)**
+    
+7: Go to **SSO** and copy the **SAML 2.0 Endpoint HTTP** and paste into the **Identity Provider URL** field in Codio, and find and copy the **X. 509 Certificate** into the **Certificate** field in Codio.
+
+.. Note:: These field names relate to onelogin and may be different in other Service Providers
```

Comment:
Line 39 - OneLogin should be cased as I just wrote and maybe bolded or italicized to indicate it is a product (not sure what you usually do). Service Providers does not need caps.